include/sys/Makefile.am

```diff
@@ -122,6 +122,7 @@ COMMON_H = \
 	$(top_srcdir)/include/sys/zio.h \
 	$(top_srcdir)/include/sys/zio_impl.h \
 	$(top_srcdir)/include/sys/zio_priority.h \
+	$(top_srcdir)/include/os/linux/zfs/sys/zpl_xattr.h \
```

Comment:
This belongs in include/os/linux/zfs/sys/Makefile.am

Reply:
Moved.